images/monitoring/Dockerfile

```diff
@@ -21,10 +21,27 @@ ENV TOOL_NAME monitoring
 # Temporarily add test-infra to the image to build custom tools
 ADD ./tools $TEMP_TOOLS_REPO_DIR
 
+# These go dependencies need to be included before cloud mail files to avoid stale checkout due to
```

Comment:
` cloud mail files to avoid stale checkout `?

Reply:
The exact error is as follows for not including these packages before mail: 
```
package google.golang.org/genproto/googleapis/rpc/status: /go/src/google.golang.org/genproto exists but /go/src/google.golang.org/genproto/.git does not - stale checkout?
make: *** [all] Error 1
```

Basically, because .git files are not present in zip folder for mail client, when running `go get`, it cannot fetch the version information and get this error.

Another attempt to get the comment message more understandable, how about: 
```
These go dependencies need to be added before cloud mail dependencies. This is due to `go get` uses git to check and update the dependent packages. Since .git files are not present in the mail package, the subsequent `go get` on the cloud.google.com/go and google.golang.org/genproto/... would fail. 
https://golang.org/cmd/go/#hdr-Download_and_install_packages_and_dependencies
```

Reply:
I just wasnt sure what we mean by stale checkout :)

I am fine with adding a simple comment like `Get cloud mail dependencies first as it is needed by the mail api` or something like that. Maybe even `Add cloud mail go dependencies first.`

Reply:
Updated.